Index: cairo-dock/src/cairo-dock-internal-labels.c
===================================================================
--- cairo-dock/src/cairo-dock-internal-labels.c	(revision 1823)
+++ cairo-dock/src/cairo-dock-internal-labels.c	(working copy)
@@ -17,6 +17,7 @@
 #include "cairo-dock-internal-dialogs.h"
 #include "cairo-dock-internal-icons.h"
 #include "cairo-dock-container.h"
+#include "cairo-dock-dock-facility.h"
 #define _INTERNAL_MODULE_
 #include "cairo-dock-internal-labels.h"
 
