--- cairo-dock-plugins-3.4.0/Dbus/interfaces/ruby/CDApplet.rb.initialization	2014-12-19 19:06:23.010243636 +0900
+++ cairo-dock-plugins-3.4.0/Dbus/interfaces/ruby/CDApplet.rb	2014-12-19 20:22:32.177163226 +0900
@@ -26,7 +26,7 @@ require 'dbus'
 require 'parseconfig'
 
 class CDApplet
-	attr_accessor :cConfFile, :cAppletName, :icon, :sub_icons, :config, :bus, :_cMenuIconId, :cParentAppName, :cBusPath
+	attr_accessor :cConfFile, :cAppletName, :icon, :sub_icons, :config, :bus, :_cMenuIconId, :cParentAppName, :cBusPath, :cRootDataDir, :cShareDataDir
 	
 	BOTTOM = 0
 	TOP    = 1
