Index: plug-ins/dnd2share/src/applet-backend-imagebin.h
===================================================================
--- plug-ins/dnd2share/src/applet-backend-imagebin.h	(revision 1816)
+++ plug-ins/dnd2share/src/applet-backend-imagebin.h	(working copy)
@@ -1,5 +1,5 @@
-#ifndef __CD_BACKEND_UPPIX__
-#define  __CD_BACKEND_UPPIX__
+#ifndef __CD_BACKEND_IMAGEBIN__
+#define  __CD_BACKEND_IMAGEBIN__
 
 #include <cairo-dock.h>
 #include "applet-struct.h"
Index: plug-ins/dnd2share/src/applet-backend-imagebin.c
===================================================================
--- plug-ins/dnd2share/src/applet-backend-imagebin.c	(revision 1816)
+++ plug-ins/dnd2share/src/applet-backend-imagebin.c	(working copy)
@@ -31,7 +31,7 @@
 	{
 		// on remplace les espaces du texte par des %20.
 		gchar **cTextParts = g_strsplit (cFilePath, " ", -1);
-		GString *sContent = g_string_new_sized (strlen (cFilePath) + 300);  // 100 espaces d'avance.
+		GString *sContent = g_string_sized_new (strlen (cFilePath) + 300);  // 100 espaces d'avance.
 		int i;
 		for (i = 0; cTextParts[i] != NULL; i ++)
 		{
Index: plug-ins/dnd2share/src/applet-init.c
===================================================================
--- plug-ins/dnd2share/src/applet-init.c	(revision 1816)
+++ plug-ins/dnd2share/src/applet-init.c	(working copy)
@@ -16,6 +16,7 @@
 #include "applet-dnd2share.h"
 #include "applet-backend-uppix.h"
 #include "applet-init.h"
+#include "applet-backend-imagebin.h"
 
 
 CD_APPLET_DEFINITION ("dnd2share",
