--- /usr/include/X11/extensions/XTest.h	2009-06-18 11:14:02.000000000 +0900
+++ usr/include/X11/extensions/XTest.h	2009-06-18 17:05:41.000000000 +0900
@@ -47,7 +47,22 @@
 
 #ifndef _XTEST_SERVER_
 
+#if 0
 #include <X11/extensions/XInput.h>
+#else
+#include <X11/Xlib.h>
+
+typedef struct {
+        unsigned char   input_class;
+        unsigned char   event_type_base;
+} XInputClassInfo;
+
+typedef struct {
+        XID                    device_id;
+        int                    num_classes;
+        XInputClassInfo        *classes;
+} XDevice;
+#endif
 
 _XFUNCPROTOBEGIN
 
